modules/ai/src/main/Config.scala

```diff
@@ -46,7 +48,8 @@ private[ai] case class Config(
     setoption("UCI_Chess960", req.variant == Chess960),
     setoption("UCI_KingOfTheHill", req.variant == KingOfTheHill),
     setoption("UCI_3Check", req.variant == ThreeCheck),
-    setoption("UCI_Horde", req.variant == Horde))
+    setoption("UCI_Horde", req.variant == Horde),
+    setoption("SyzygyProbeLimit", if (req.variant == Standard || req.variant == Chess960) 6 else 0))
```

Comment:
@ddugovic Here we disable tablebases for non-standard variants on the Scala side of things.